renpy/config.py

```diff
@@ -1264,6 +1264,9 @@ def say_attribute_transition_callback(*args):
 # the style inspector.)
 alternate_unelide_path = None
 
+# An alias -> font map.
+font_names = {}
```

Comment:
Perhaps a good name would be font_name_map. Font_names doesn't say anything about what it does - I'd expect this to be a list or set.